src/components/Delegation/DelegationCenter.stories.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React from 'react'
+
+import {storiesOf} from '@storybook/react-native'
+
+import DelegationCenter from './DelegationCenter'
+
+storiesOf('DelegationCenter', module).add(
```

Comment:
FYI they suggest moving away from `storiesOf` and using their new "CSP" pattern instead when possible

Reply:
Yep, but it's not available for react-native yet.